en/install.rst

```diff
@@ -46,8 +50,12 @@ developing with Zephir.
 
 Installing Zephir
 -----------------
```

Comment:
`s/that/the/` or `s/that/that the/`.

`s/following tutorial_/by following its tutorial_\ ./` or similar.